website/app/error.tsx

```
@@ -0,0 +1,30 @@
+            <div className="relative z-10 mx-auto max-w-md px-6 text-center">
+                <h1 className="mb-4">
+                    <span className="text-[--color-text]">Something went </span>
+                    <span className="text-gradient-static">wrong</span>
+                </h1>
+                <p className="mb-8 text-[--color-text-secondary]">
+                    An unexpected error occurred. Please try again.
+                </p>
+                <button
+                    onClick={reset}
+                    className="glow-btn inline-flex items-center gap-2 rounded-xl bg-gradient-to-r from-blue-600 to-blue-500 px-6 py-3 text-sm font-semibold text-white transition-all hover:from-blue-500 hover:to-cyan-500"
+                >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)